CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ MESSAGE( STATUS "CMAKE_CL_64: ${CMAKE_CL_64}")
 SET(CMAKE_CXX_STANDARD 11)
 SET(CMAKE_CXX_STANDARD_REQUIRED ON)
 SET(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Based on https://cmake.org/cmake/help/v3.15/command/add_library.html
I would assume this is not necessary.

Reply:
You are correct, I did not realize that we were building SUPRA statically.....